Assets/Scripts/Constants/Constants.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace Constants
+{
+    public static class Enemy
```

Comment:
Besser wäre hier glaube ich eine Klasse mit dem Namen "Consts" und da drin sind dann alle constanten
Zum Beispiel: Consts.ENEMY_MAX_NAVMESH_MAPPING_DISTANCE.
Wenn du unbedingt Gruppieren willst und das ENEMY_* nicht reicht/blöd ist, könntest du auch in der Klasse eine Unterklasse machen: Consts.Enemy.MAX_NAVMESH_MAPPING_DISTANCE

Reply:
done